physical/raft/raft.go

```diff
@@ -851,6 +852,9 @@ type SetupOpts struct {
 	// RecoveryModeConfig is the configuration for the raft cluster in recovery
 	// mode.
 	RecoveryModeConfig *raft.Configuration
+
+	// overrideMsgpackUseNewTimeFormat is used for testing backwards compatability
+	overrideMsgpackUseNewTimeFormat *bool
```

Comment:
What's the motivation here, if Vault has never used the old format?

Reply:
I guess it's to allow TestRaftNetworkClusterWithMultipleTimeEncodingsSet to exist, but see my comment on that.